contrib/easage-pack.cmd

```diff
@@ -0,0 +1,52 @@
+@echo off
+rem Contributor: Teteros
+rem Non-alphanumeric filenames were untested and might cause problems
+
+title Drag'n'Drop a selection onto this script to pack into BIGs
+rem Usage: Drag'n'Drop files and folders onto easage-pack.cmd
+rem to pack into BIG format files
+rem easage.exe 'pack' archives recursively
+rem i.e sub-directories are also packed
+if not exist easage.exe goto FileMissing
+
+rem Use wmic command to get current date information in YYYYMMDD format
+for /f %%# in ('wMIC Path Win32_LocalTime Get /Format:value') do @for /f %%@ in ("%%#") do @set %%@

```

Comment:
Is this marker (`^M`) going to cause issues for anyone?

Reply:
`^M` is `\r`
It's Windows `\r\n` endings, aka `CR`+`LF`, while Linux only needs `LF`

I've added them in in case someone wants to quickly edit the `.cmd` file in Windows' Notepad. If you omit the `CR`'s notepad will just show everything on one continuous line which isn't human readable.

Why github only shows that marker on that line I'm not sure, whole file has them on my editor.